### PR TITLE
Show the admin UI button if the user is an admin

### PR DESCRIPTION
**Pull request by ChrisRzech**
### Key features
* Checks if the last login type was an admin; if it is, show the admin UI button.

### Future Improvements
* none

### Notes
none

### CODE REVIEW

**Comment by jt8483** [comment]
wow nice !

File: src/windows/mainwindow.cpp [hunk 1]
```diff
@@ -125,7 +126,7 @@ void MainWindow::changeNavState(ViewStates state)
         m_navbar->item(0)->setHidden(false);
         m_navbar->item(1)->setHidden(false);
         m_navbar->item(2)->setHidden(false);
-        m_navbar->item(3)->setHidden(false);
+        m_navbar->item(3)->setHidden(Login::getType() == Login::Type::ADMIN ? false : true);
```
Comment (typokign) [inline src/windows/mainwindow.cpp:129]: `Login::getType() != Login::Type::ADMIN`

File: src/windows/mainwindow.cpp [hunk 2]
```diff
@@ -137,7 +138,7 @@ void MainWindow::changeNavState(ViewStates state)
         m_navbar->item(0)->setHidden(true);
         m_navbar->item(1)->setHidden(true);
         m_navbar->item(2)->setHidden(true);
-        m_navbar->item(3)->setHidden(true);
+        m_navbar->item(3)->setHidden(Login::getType() == Login::Type::ADMIN ? true : false);
```
Comment (typokign) [inline src/windows/mainwindow.cpp:141]: `Login::getType() == Login::Type::ADMIN`

**Review by typokign — CHANGES_REQUESTED** [review]
cmon kris

**Comment by ChrisRzech** [comment]
PR will be outdated soon, closing.

**Pull request closed without merging**